docs/architecture/governance/improvement-process.md

```diff
@@ -5,49 +5,54 @@ last_update:
 ---
 As Tezos is a large decentralized project that is constantly evolving, its members need to have a mechanism for proposing improvements to the ecosystem.
 
-A Tezos Improvement Proposal (or TZIP, pronounced "tee-zip") is a document that offers ways to improve Tezos via new features, tools, or standards (e.g. smart contract specifications).
+A Tezos Improvement Proposal (or TZIP, pronounced "tee-zip") is a document that offers ways to improve Tezos via new features, tools, or standards (e.g. smart contract interface specifications).
```

Comment:
Don't use Latinate abbreviations like "e.g."

Reply:
Style guides generally recommend against Latinate abbreviations like e.g. and i.e but it's not a big deal.

Reply:
Of course, the importance is limited, but I'm asking this because I couldn't find this kind of recommendation in the Chicago or AP guides.
Thanks!